0.2.0/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM openjdk:8-jdk
+
+ENV JANUS_VERSION=0.2.0 \
+    JANUS_HOME=/opt/janusgraph \
+    JANUS_CONFIG_DIR=/etc/opt/janusgraph \
+    JANUS_DATA_DIR=/var/lib/janusgraph \
+    JANUS_INDEXES="default" \
+    JANUS_INDEX_DEFAULT_BACKEND="lucene" \
+    JANUS_INDEX_DEFAULT_DIRECTORY=/var/lib/janusgraph/index \
+    JANUS_STORAGE_BACKEND=berkeleyje \
```

Comment:
I'm retracting my stance from the dev thread on suggesting inmemory. Since everything is being packaged together as a giant zip there's no added cost in using what's already being downloaded and berkeley makes more sense than inmemory.